ml-agents/tests/yamato/scripts/run_gym.py

```diff
@@ -34,8 +34,26 @@ def main(env_name):
         env.close()
 
 
+def test_closing(env_name):
+    """
+    Run the gym test and closes the environment multiple times
```

Comment:
is this to make sure an exception doesn't get thrown?

Reply:
I think because we do just a try and finally (with no catch), if there is an exception it run the finally then re-raise the exception.

Reply:
```
$ cat test_exit.py 
def main():
    try:
        print("trying")
        raise RuntimeError("oops")
    finally:
        print("cleaning up")
    print("unreachable!")

if __name__ == "__main__":
    main()

$ python test_exit.py 
trying
cleaning up
Traceback (most recent call last):
  File "test_exit.py", line 10, in <module>
    main()
  File "test_exit.py", line 4, in main
    raise RuntimeError("oops")
RuntimeError: oops
```

Reply:
got it